.github/workflows/validate-json-schema.yml

```diff
@@ -0,0 +1,20 @@
+name: Validate JSON Schema
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  validate-json:
```

Comment:
We already have JSON linter: https://github.com/actions/runner-images/blob/b0de0a7049c04caed4a5e181fc29a1883ce37797/.github/workflows/linter.yml#L29C11-L29C24

I am surprised that it doesn't throw on invalid JSON files

Reply:
That's bad naming from me, this is validating the json schema matches in `./schema/toolset-schema.json` rather than validating the json, basically another layer on top, yeah the file is valid json but does it meet the patterns and rules in the json schema.

Reply:
I've renamed the job to make this clearer